doc/CONFIGURATION.md

```diff
@@ -6,6 +6,7 @@ In most scenarios, you can use Mountpoint by running the following command, wher
 
 We've tried hard to make this simple command adopt good defaults for most scenarios. However, some scenarios may need additional configuration. This document shows how to configure these elements of Mountpoint:
 * [AWS credentials](#aws-credentials)
+* [Caching configuration](#caching-configuration)
```

Comment:
Should be below bucket configuration, and probably below file system configuration too. Also needs a sentence description (caching what?).